types/rifat__miio/rifat__miio-tests.ts

```diff
@@ -0,0 +1,9 @@
+import { device } from '@rifat/miio';
+
+device({ address: '192.168.0.1' }); // $ExpectError
```

Comment:
The docs suggest this should be valid if the token can be determined automatically or from storage

Reply:
Will be resolved